### PR TITLE
Bug 1184180 - Implement screen shade feature for screen reader.

### PR DESCRIPTION
**Pull request by eeejay**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40a9933bcfc3f3771466e502467befd5a5e42fdf

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6fb7d911e2b192feb60aa306ce9a5365a4ef6f10

File: apps/system/js/screen_manager.js [hunk 1]
```diff
@@ -333,6 +342,21 @@
...
+      this._screenShade = true;
+      this.setScreenBrightness(0, true);
+    },
+
+    turnShadeOff: function scm_turnShadeOff() {
+      // Bug 1185036: If the screen was disable with the shade on, we won't
+      // the screen won't turn on again until the user puts their finger on the
+      // screen again.
+      var brightness = parseFloat(this._savedBrightness) ||
+        parseFloat(this._userBrightness) || 1;
```
Comment (timdream) [inline apps/system/js/screen_manager.js]: Remove these `parseFloat()`. It's not possible these value can be anything other than numbers.

Comment (eeejay) [inline apps/system/js/screen_manager.js]: I encountered them being strings in some cases. I think.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=df7d4f909db1f264f4b1a881917d1e06729515d1

**Pull request merged**